src/provider.ts

```diff
@@ -20,24 +20,32 @@ import {
   NodeProvider,
```

Comment:
https://github.com/alephium/walletconnect/pull/17/files#diff-e0e6d91455eef50c5793e300039e5cdc7e6d372707172b639564495e4fa0a2f3R200 is the only actual change in this file, the rest is formatting